code/game/machinery/doors/airlock.dm

```diff
@@ -72,6 +72,7 @@
 	var/hasShocked = 0 //Prevents multiple shocks from happening
 	autoclose = 1
 	var/obj/item/device/doorCharge/charge = null //If applied, causes an explosion upon opening the door
+	var/obj/item/weapon/note //Any papers pinned to the airlock
```

Comment:
>showing or having an insensitive and cruel disregard for others.

What's callous about explaining that it's your fault you're in the situation you are in, and thus complaining to me is pointless?